grunttasks/concat.js

```diff
@@ -7,6 +7,10 @@ module.exports = function (grunt) {
     requirejs: {
       dest: '<%= yeoman.tmp %>/scripts/main.js',
       src: ['app/bower_components/requirejs/require.js', '<%= yeoman.tmp %>/scripts/main.js']
+    },
+    easteregg: {
```

Comment:
@vladikoff - I wanted the easteregg to live as a standalone module so that can be included by other cloud services projects. 

To meet the "prod only" constraint, I initially had @TDA use an initialization function that only displayed the easter egg if the page's origin was not localhost. 

This didn't seem right because it meant we wouldn't ever be able to test locally, even if running production resources. Adding a new item to `/config` felt like it is well out of scope for this.

Since the easter egg is outside of the core scope to everything else we do, and can can run as a stand alone module, I suggested we just use the build system to package it during build time.